unicodetools/org/unicode/tools/emoji/candidateData.txt

```diff
@@ -111,19 +112,19 @@ Proposal=L2/19-378
 After=🤭
 U+1FAE3
 Name=face with peeking eye
-Keywords=can’t look away | captivated | staring | peep
+Keywords=captivate | peep | stare
```

Comment:
I think the change in tense for “captivated” is wrong, this is about the captivatee not the captivator.